SparkleSpin/PlayerListViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PlayerListViewModel.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 2/27/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+class PlayerListViewModel {
+    
+    private let playerListModel: PlayerListModel
+    var nameString: String? = nil
+    
+    init(playerListModel: PlayerListModel) {
+        self.playerListModel = playerListModel
+    }
+    
+    func createPlayerWith(name: String) -> PlayerModel {
+        return PlayerModel(name: name)
+    }
+    
+    func addPlayerToPlayerList(player: PlayerModel) {
+        playerListModel.playerList?.append(player)
+    }
+    
+    func getPlayerList() -> [PlayerModel] {
+        return playerListModel.playerList ?? []
+    }
+    
+    func deletePlayer(playerToDelete: PlayerModel) {
+        for (index, player) in playerListModel.playerList?.enumerated() ?? [].enumerated() {
+            if player == playerToDelete {
+                playerListModel.playerList?.remove(at: index)
+            }
+        }
+    }
```

Comment:
For loops are always a good way to do things, I would suggest to play a little with `array.index(of: itemToRemove)` and `array.filter` to say other possible ways to do this same thing, just for practice, you don't have to do it for this PR.

Reply:
Going to try my hand at the `.filter`. I'm trying to incorporate high order functions when I can.